src/EditorFeatures/Core.Wpf/InlineDiagnostics/InlineDiagnosticsTaggerProvider.cs

```diff
@@ -29,7 +29,7 @@ namespace Microsoft.CodeAnalysis.Editor.InlineDiagnostics
     [Export(typeof(ITaggerProvider))]
     [ContentType(ContentTypeNames.RoslynContentType)]
     [TagType(typeof(InlineDiagnosticsTag))]
-    internal class InlineDiagnosticsTaggerProvider : AbstractDiagnosticsAdornmentTaggerProvider<InlineDiagnosticsTag>
+    internal sealed class InlineDiagnosticsTaggerProvider : AbstractDiagnosticsAdornmentTaggerProvider<InlineDiagnosticsTag>
```

Comment:
i like to seal inheritance chains to make it so that as much as possible you just have abstracts in teh middle with abstract functionality, and then sealed leaves which define the behavior which is no longer changeable.